index.js

```diff
@@ -3656,6 +3656,42 @@
       [Fn(a, b), m(a), m(a)],
       Z.sortBy);
 
+  //# zipWith :: (Applicative m, Foldable m, Monoid m) => (a -> b -> c) -> m a -> m b -> Arrray c
+  //.
+  //. Creates a new array out of the two supplied structures
+  //. by applying the given predicate to each equally-positioned
+  //. pair of elements from each given [Foldable][]. The returning
+  //. array is truncated to the length of the shorter of the two
+  //. input Foldables.
+  //.
+  //. ```javascript
+  //. > S.zipWith(S.add, [1, 2, 3], [3, 2, 1])
+  //. [4, 4, 4]
+  //.
+  //. > S.zipWith(a => b => [a, b], [1, 3, 5], [2, 4, 6])
+  //. [[1, 2], [3, 4], [5, 6]]
+  //.
+  //. > S.zipWith(a => b => Cons(a, Cons(b, Nil)), Cons('foo', Nil), Cons('bar', Nil))
+  //. [Cons('foo', Cons('bar', Nil))]
+  //. ```
+  function zipWith(pred, a, b) {
```

Comment:
A value of type `a -> b -> c` is not a predicate, so `pred` is a confusing identifier. Let's use `f` instead. :)

Reply:
thanks!